aws-msk/dashboards/aws-msk.json

```diff
@@ -0,0 +1,314 @@
+{
+  "name": "Amazon - MSK Monitoring",
+  "description": null,
+  "permissions": "PUBLIC_READ_WRITE",
```

Comment:
You'll need to remove this line, these are rewritten when a user install a quickstart